docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1248,27 +1358,47 @@ async function startScreenShare() {
     });
 
     meetingStreamsLocalShareVideo.srcObject = localShareVideoStream.outputStream;
+    meetingStreamsLocalShareAudio.srcObject = localShareAudioStream.outputStream;
```

Comment:
when sharing without audio the code crashes here, so end up not enabling the "stop share" button

Reply:
fixed it